SingularityUI/app/application.coffee

```diff
@@ -23,45 +23,44 @@ class Application
     initialize: ->
         @setupGlobalErrorHandling()
 
-        @setupUser()
+        @setupUser =>
```

Comment:
The changes here are just changes in indentation - rendering of pages will be delayed until we tried to fetch the user so that we can avoid awkward situations where the admin dropdown in the toolbar disappears or suddenly appears on the second loaded page.